tasks/section_5_Access_Authentication_and_Authorization.yaml

```diff
@@ -621,8 +621,8 @@
 - name: 5.4.4 Ensure password hashing algorithm is SHA-512
   lineinfile:
     dest: /etc/pam.d/common-password
-    regexp: '^password\s+\[success'
-    line: "password [success=1 default=ignore] pam_unix.so sha512"
+    regexp: '^password\s+\[\s{0,}(success=[1-9])\s+.+\s{0,}\]\s+(.*)\s+(md5|blowfish|bigcrypt|sha256|sha512)\s+?(.*)'
+    line: 'password [\1 default=ignore] \2 sha512 \4'
```

Comment:
Why do you want to add the `default=ignore` option? Maybe it's better to just add the `sha512` and leave what's between the brackets alone?

Reply:
You're right, I'm updating the PR right now